<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <!-- Hardware Control File -->
  <include file="$(find robomuse_hardware)/launch/hw_control.launch"/>

  <!--Conversion from Quaternions to Euler-->
  <node pkg="robomuse_hardware" name="converter" type="euler.py" />

  <!--Filtered Theta-->
  <node pkg="robomuse_hardware" name="fuser" type="fuser.py" />
  
  <!--PS3 Teleop Launch-->
  <include file="$(find teleop_twist_joy)/launch/teleop.launch"/>

  <!--Keyboard Teleop Launch-->
  <node pkg="robomuse_simulation" name="robomuse5_teleop" type="teleop.py" />

  <!--Teleop Mux Launch-->
  <include file="$(find twist_mux)/launch/twist_mux.launch"/>

  <!--rqt_image_view-->
  <node pkg="rqt_image_view" type="rqt_image_view" name="image_view_debug">

</launch>
